backend/openapi.json

```diff
@@ -663,6 +663,7 @@
         "required": [
           "id",
           "name",
+          "location",
```

Comment:
Please update the frontend openapi file as well and make sure all election definitions there are updated as well.